### ISSUE TITLE
2.1.2 No Keyboard Trap

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4183]
Keyboard-only users must be able to navigate to and from all parts of a website. No mouse should be necessary to move away from any element of a website.

With no keyboard trap, it’s especially important to test:

- [x] pop-ups
- [x] forms 
- [ ] embeds

on the platform to make sure that users can back out of them with ease.

Related issues: #2300
